lib/zlib/deflate.c

```diff
@@ -34,6 +34,9 @@ typedef struct {
 static bool
 feed(jose_io_t *io, const void *in, size_t len, typeof(deflate) *func)
 {
+    if (len > MAX_COMPRESSION_SIZE) {
```

Comment:
this seems to prevent both operations of compression and decompression ... not ideal but if the size is big enough it is probably ok

Reply:
You are right. I will try limiting just the decompression